### PR TITLE
Fix multiple definition

### PR DESCRIPTION
**Pull request by Jamaika1**

### CODE REVIEW

**Comment by mpeg5** [comment]
> _No description provided._

In order to understand PR description is required.
"Patch 1" Subject is not enough.

File: src_base/xevd_ipred.c [hunk 1]
```diff
@@ -676,7 +676,7 @@ void xevd_ipred_uv_b(pel *src_le, pel *src_up, pel *src_ri, u16 avail_lr, pel *d
     }
 }
 
-int intra_mode_list[IPD_CNT] = {
+/*int intra_mode_list[IPD_CNT] = {
```
Comment (mpeg5) [inline src_base/xevd_ipred.c:679]: Can you remove this "intra_mode_list" from here instead of commenting?

File: src_base/xevd_ipred.h [hunk 2]
```diff
@@ -60,5 +60,5 @@ void ipred_vert(pel *src_le, pel *src_up, pel * src_ri, u16 avail_lr, pel *dst,
 void ipred_plane(pel *src_le, pel *src_up, pel *src_ri, u16 avail_lr, pel *dst, int w, int h, int bit_depth);
 void ipred_bi(pel *src_le, pel *src_up, pel *src_ri, u16 avail_lr, pel *dst, int w, int h, int bit_depth);
 void ipred_ang(pel *src_le, pel *src_up, pel *src_ri, u16 avail_lr, pel *dst, int w, int h, int ipm, int bit_depth);
-int  intra_mode_list[IPD_CNT];
+//int  intra_mode_list[IPD_CNT];
```
Comment (mpeg5) [inline src_base/xevd_ipred.h:63]: could you remove "intra_mode_list" line?

File: src_main/xevdm_ipred.c [hunk 3]
```diff
@@ -303,6 +303,18 @@ void xevdm_ipred_uv(pel *src_le, pel *src_up, pel *src_ri, u16 avail_lr, pel *ds
     }
 }
 
+int intra_mode_list[IPD_CNT] = {
```
Comment (mpeg5) [inline src_main/xevdm_ipred.c:306]: can you add "static" in front of this variable? if it is used only for this code file.

**Review by mpeg5 — CHANGES_REQUESTED** [review]

File: src_main/xevdm_ipred.c (hunk 3 above)
Comment (Jamaika1) [reply src_main/xevdm_ipred.c]: I add "static".

File: src_base/xevd_ipred.h (hunk 2 above)
Comment (Jamaika1) [reply src_base/xevd_ipred.h]: I remove "intra_mode_list".

File: src_base/xevd_ipred.c (hunk 1 above)
Comment (Jamaika1) [reply src_base/xevd_ipred.c]: I remove "intra_mode_list".

**Review by mpeg5 — APPROVED** [review]
LGTM

**Pull request merged**